Multisig/Cross-layer/App.swift

```diff
@@ -23,11 +23,12 @@ class App {
     // Data Layer
 
     // Services
-    var safeRelayService = SafeRelayService(url: URL(string: "https://safe-relay.rinkeby.gnosis.io")!,
-                                            logger: LogService.shared)
+    var safeRelayService = SafeRelayService(
+        url: URL(string: "https://safe-relay.rinkeby.gnosis.io")!, logger: LogService.shared)
     
-    var safeTransactionService = SafeTransactionService(url: URL(string: "https://safe-transaction.rinkeby.gnosis.io")!,
-    logger: LogService.shared)
+    var safeTransactionService = SafeTransactionService(
+        url: URL(string: "https://safe-transaction.rinkeby.gnosis.io")!, logger: LogService.shared)
 
-    var nodeService = EthereumNodeService(url: URL(string: "https://rinkeby.infura.io/v3/438e11915f8b4834a05e7810b88db4b3")!)
+    var nodeService = EthereumNodeService(
+        url: URL(string: "https://rinkeby.infura.io/v3/438e11915f8b4834a05e7810b88db4b3")!)
```

Comment:
you know what, I've added this when adding ENS, and now our API key is published here :(

Reply:
damn, we need to invalidate it